packages/core/src/snapshot.js

```diff
@@ -110,6 +110,7 @@ function getSnapshotOptions(options, { config, meta }) {
       allowedHostnames: config.discovery.allowedHostnames,
       disallowedHostnames: config.discovery.disallowedHostnames,
       networkIdleTimeout: config.discovery.networkIdleTimeout,
+      devicePixelRatio: config.discovery.devicePixelRatio,
```

Comment:
* add unit tests in `packages/core/test/unit/snapshot.test.js`
* update desc.
* Create new Github issue DPR fix `cli-snapshot`, with possible solution if any.